tests/testthat/test-retrieve.R

```diff
@@ -233,10 +237,25 @@ test_that("explicit path to binary packages work", {
 
   skip_if_not(renv_platform_macos())
 
+  renv_tests_scope()
+  renv_scope_local()
+
+  # make sure we have a binary package in the cellar to test with
+  srcpath <- renv_tests_path("local/skeleton/skeleton_1.0.1.tar.gz")
+  binpath <- renv_tests_path("local/skeleton/skeleton_1.0.1.tgz")
+  defer(unlink(binpath))
+
+  # create the binary
```

Comment:
The old binary was actually built with an older version of R, and so would fail to load when the test suite was run with a newer version of R.